Automattic/wp-calypso
# Domain Management i1 - Show domains with fields that will not change as part of bulk contact info editing

Domains may have different restrictions in terms of what fields can be edited during contact update, eg. `.fr` TLDs do not allow editing of name fields. This PR adds a notice to the bulk contact edit form to let the user know that some fields for some domains will not be affected the edit.

Closes https://github.com/Automattic/dotcom-forge/issues/3654

## Proposed Changes

* check domains for whois unmodifiable fields
* show notice for domains with unmodifiable fields

## Testing Instructions


Register a `.fr` domain as it doesn't allow editing of certain fields.
* browse `domains/manage?flags=domains/bulk-actions-contact-info-editing`
* Select multiple ccTLDs, eg. .`uk, .ca, .fr ` and click `Edit contact information`
* Verify the following notice is shown
![image](https://github.com/Automattic/wp-calypso/assets/47489215/b277a8d0-df89-4384-b148-ac89b916a0ea)
